src/pages/_app.tsx

```diff
@@ -8,6 +9,14 @@ import './style.css'
 export default function App({ Component, pageProps }) {
   useAnchorLinkAnalytics()
 
+  if (typeof window !== 'undefined' && process.env.AXE_ENABLED) {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const ReactDOM = require('react-dom')
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const axe = require('@axe-core/react')
+    axe(React, ReactDOM, 1000)
+  }
```

Comment:
FWIW, outside of the component body seems like the correct spot.

Reply:
I wasn't sure! Before I started making changes today, the bundle analysis said the bundle size decreased. I thought maybe moving it outside of the `App` component was the problem, but I'm seeing no change in the bundle analysis between those two approaches. I'll move it back out because I agree that seems like the better place for it!

Reply:
@BRKalow I think what Dylan uncovered this week explains why the bundle size stuff was weird! The assembly branch's artifact was not staying up to date. The comment updates accurately now, no increase to the bundle size! 👏🏻